src/System.Drawing.Primitives/src/System/Drawing/Point.cs

```diff
@@ -257,19 +203,10 @@ public void Offset(Point p)
         ///       string.
         ///    </para>
         /// </summary>
-        public override string ToString()
-        {
-            return "{X=" + X.ToString() + ",Y=" + Y.ToString() + "}";
-        }
+        public override string ToString() => $"{{X={X},Y={Y}}}";
```

Comment:
We'd need to measure to know whether this is a positive change performance-wise; regardless, though, this is less efficient than ```$"{X={X.ToString()},Y={Y.ToString()}}}", as the version you have incurs boxing for both X and Y.